### PR TITLE
Update static/AudioPlayerto use ../shared/Audio instead of the native `<audio>` for Wasm support

### PR DESCRIPTION
**Pull request by whitphx**
## Description

I missed `js/audio/static/AudioPlayer.svelte` in #5627 which also should be converted to be Wasm-compatible.

Closes: #5999

### CODE REVIEW

File: js/audio/static/AudioPlayer.svelte [hunk 1]
```diff
@@ -81,24 +82,21 @@
...
-		use:loaded={{ autoplay }}
-		controls
-		preload="metadata"
-		src={value?.data}
-		on:play
-		on:pause
-		on:ended={handle_ended}
-		data-testid={`${label}-audio`}
-	/>
+	<div class="container">
```
Comment (whitphx) [inline js/audio/static/AudioPlayer.svelte:85]: I added this container div for the consistency to `/js/audio/interactive/Audio.svelte`:
https://github.com/gradio-app/gradio/blob/584fd97d599540c2770377fc7aecf08619f99cc8/js/audio/interactive/Audio.svelte#L298
while it has no style now thanks to https://github.com/gradio-app/gradio/pull/5934

**Review by aliabd — APPROVED** [review]
Nice! Works as expected. Thanks @whitphx

File: js/audio/static/AudioPlayer.svelte
[the remark this answers is not in the document]
Comment (whitphx) [reply js/audio/static/AudioPlayer.svelte:1]: This is needed to make this sample work on Wasm (this sample is included in the playground as "Tone" demo):
In #5627 I was only thinking about Examles, but it was not enough.

```python
import numpy as np
import gradio as gr

notes = ["C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A", "A#", "B"]

def generate_tone(note, octave, duration):
    sr = 48000
    a4_freq, tones_from_a4 = 440, 12 * (octave - 4) + (note - 9)
    frequency = a4_freq * 2 ** (tones_from_a4 / 12)
    duration = int(duration)
    audio = np.linspace(0, duration, duration * sr)
    audio = (20000 * np.sin(audio * (2 * np.pi * frequency))).astype(np.int16)
    return sr, audio

demo = gr.Interface(
    generate_tone,
    [
        gr.Dropdown(notes, type="index"),
        gr.Slider(4, 6, step=1),
        gr.Textbox(value=1, label="Duration in seconds"),
    ],
    "audio",
)
if __name__ == "__main__":
    demo.launch()
```

[the remark this answers is not in the document]
Comment (pngwn) [reply js/audio/static/AudioPlayer.svelte:1]: Ah cool.

**Pull request merged**